run.py

```diff
@@ -1039,10 +1040,7 @@ def setting_post(
 
     root_dir = args.voicevox_dir if args.voicevox_dir is not None else engine_root()
 
-    if not USER_SETTING_PATH.is_file():
-        setup_setting_file()
-
-    setting_file_path = args.setting_file if args.setting_file else USER_SETTING_PATH
+    setting_file_path = args.setting_file
 
     setting_loader = SettingLoader(setting_file_path)
```

Comment:
好みですが、せっかくシンプルなのでこうでも良いかも！！
```suggestion
    setting_loader = SettingLoader(args.setting_file)
```